source/scripts/enum.lua

```diff
@@ -14,6 +14,6 @@ return {
     basetypeBuilding1 = 7,
     basetypeBuilding2 = 8,
     baseMaxFuel = 15,
-	
-	
+
+
 }

```

Comment:
Why did you add two new lines here? Personally I prefer writing stuff clean and compact as much as possible.

Reply:
I didn't add new lines, I removed the whitespace (tabs)